app/cdap/components/WranglerGrid/SelectColumnPanel/DataTable/CheckboxInput/index.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import { Checkbox, FormControlLabel } from '@material-ui/core';
+import { ICheckBoxInputProps } from 'components/WranglerGrid/SelectColumnPanel/DataTable/types';
+
+export default function({
+  selectedColumns,
+  handleDisableCheckbox,
+  columnDetail,
+  onMultipleSelection,
+  label,
+  columnIndex,
+}: ICheckBoxInputProps) {
+  const disabled =
```

Comment:
again if you're looking for one item don't use filter and then check its length.

Reply:
Converted ternary to if else and replaced filter with findIndex and also switched this file code to InputWidgets as in few cases logic was same so to reuse it

Reply:
we have changed filter to some